apps/yapms/src/lib/components/modals/editcandidatemodal/EditCandidateModal.svelte

```diff
@@ -23,6 +23,25 @@
 		newColors = newColors.slice(0, newColors.length - 1);
 	}
 
+	function removeCandidate() {
+		$CandidatesStore = $CandidatesStore.filter((candidate) => candidate.id !== id);
+		$SelectedCandidateStore = $TossupCandidateStore;
```

Comment:
```suggestion
		if ($SelectedCandidateStore.id === id) $SelectedCandidateStore = $TossupCandidateStore;
```
This is the only potential improvement I can pinpoint, making it so selected candidate only changes to Tossup when the selected candidate is the one being deleted.

Reply:
Good idea, fixed!